www/main/navigator/shabad/ShabadContent.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/no-cycle */
```

Comment:
@theamanjs veerji, why do we need to disable this linting rule? Couldn't find any new import with this PR.

Other than that, looks good to me.

Reply:
This is an old issue veerji, this can be done in separate issue https://github.com/KhalisFoundation/sttm-desktop/issues/1794